docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2316,9 +2315,8 @@ survey iterations.
 
 The ACS will be used for V&V testing. It is defined as: 
 
-- Sample rate of 12,000 households nationwide  
-- Sample will be stratified by state to ensure an even distribution 
-- Sample at each time step (approximates monthly)
+- To appoximate a monthly sample rate of 12,000 households nationwide with using 8 day timesteps, we will sample (12,000 * 12)/(365.25/28) = 11039 households every 28 days 
+- Note: to avoid rerunning the simulation, please oversample by 2x to allow for duplication 
```

Comment:
I don't think this is allowing for duplication, since that wouldn't require any additional rows -- perhaps you mean improper inclusion? This is a lot for that purpose but it makes sense to me to be overly generous rather than risk not sampling enough.

Reply:
Yeah, you're right I do mean that. Really it is just if we want to change something later on in any way. And I agree, it is wayyyy more than we probably need but oh well.